packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -65,6 +102,8 @@ async function getDataOutMetadata(
       answer: buildAnswerMetadatum(fieldData),
       fieldType: { isHidden: true },
       order: { isHidden: true },
+      answerArray:
+        fieldData.answerArray && buildAnswerArrayMetadatum(fieldData),
```

Comment:
nit: I feel like we can make use of mutual-exclusivity of `answer` and `answerArray` to make this bit more concise. But not important to do right now.